.drone.yml

```diff
@@ -80,372 +77,340 @@ steps:
     name: check-deprecated-apis-nginx
     environment:
       KUBERNETES_MANIFESTS: nginx.yml
-
 ---
-name: e2e-kubernetes-1.25
 kind: pipeline
 type: docker
-
-depends_on:
-  - policeman
-
-node:
-  runner: internal
-
+name: E2E Tests on kind clusters version 1.26.6
 platform:
   os: linux
   arch: amd64
-
+depends_on:
+  - policeman
+clone:
+  depth: 1
 trigger:
   ref:
     include:
-      - refs/heads/master
       - refs/heads/main
       - refs/tags/**
-
+volumes:
+  - name: dockersock
+    host:
+      path: /var/run/docker.sock
+  - name: tmp-kind
+    temp: {}
```

Comment:
This volume is not being used.
```suggestion
```

Reply:
Thanks for the recomendation, the volumen was removed.